src/pages/overview/pdf-services-api/howtos/replace-pages.md

```diff
@@ -33,55 +33,59 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
        public static void main(String[] args) {
     
            try {
-               // Initial setup, create credentials instance.
-               Credentials credentials = Credentials.servicePrincipalCredentialsBuilder()
-                    .withClientId("PDF_SERVICES_CLIENT_ID")
-                    .withClientSecret("PDF_SERVICES_CLIENT_SECRET")
-                    .build();
+                // Initial setup, create credentials instance
+                Credentials credentials = new ServicePrincipalCredentials(
+                        System.getenv("PDF_SERVICES_CLIENT_ID"),
+                        System.getenv("PDF_SERVICES_CLIENT_SECRET"));
     
-               // Create an ExecutionContext using credentials and create a new operation instance.
-               ExecutionContext executionContext = ExecutionContext.create(credentials);
-               ReplacePagesOperation replacePagesOperation = ReplacePagesOperation.createNew();
+                // Creates a PDF Services instance
+                PDFServices pdfServices = new PDFServices(credentials);
     
-               // Set operation base input from a source file.
-               FileRef baseSourceFile = FileRef.createFromLocalFile("src/main/resources/baseInput.pdf");
-               replacePagesOperation.setBaseInput(baseSourceFile);
+                // Creates an asset from source file and upload
```

Comment:
same , for full file

Reply:
As discussed, IG this is from already reviewed samples.